.github/workflows/reusable-docs.yml

```diff
@@ -1,29 +1,8 @@
-name: Docs
+name: 📝
```

Comment:
```suggestion
name: Docs
```

Reply:
(I find emoji harder to recognise at a glance, especially when I'm switching contexts -- words are much more explicit!)

Reply:
This is specifically done to make the UI displayed things shorter. So I'll wait for other people's feedbacks to see if it makes sense to revert to the previous name.
The thing is that with reusable workflows, the strings will be longer. So I kept the internal job name, but not the outer layer things.

Reply:
Let's keep "Docs", it's short enough already.

By the way, we usually don't force push to https://github.com/python/ repos, it makes it a bit harder to review, and everything gets squash merged anyway. Thanks!

PS it was great to meet you at PyCon!

Reply:
Please confirm that you are fine seeing a multi-segment thing like `Tests / Docs / Docs` in the UI. I'm not sure you fully understand the implication of how this influences the check names.

Reply:
> By the way, we usually don't force push to https://github.com/python/ repos, it makes it a bit harder to review, and everything gets squash merged anyway.

Yes, sometimes it makes it easier but I'll need to rebase after the separate PR is merged anyway. Plus it makes me easier to smuggle a nice commit message into the resulting thing.

Reply:
> Please confirm that you are fine seeing a multi-segment thing like `Tests / Docs / Docs` in the UI. I'm not sure you fully understand the implication of how this influences the check names.

I think I'd prefer `Tests / Docs / Docs` in the list of checks at the bottom if it means we can avoid this, when you click through to view the results of the checks:

![6E362A53-9461-4E8B-89B8-8037B1770EE9](https://user-images.githubusercontent.com/66076021/235082176-10072374-650e-4712-9453-4e47be48d0f5.jpeg)

Reply:
Oh, so you're talking about a different view. That's fair. I'd like to hear from others and @llanga but now I at least understand where this is coming from...

Reply:
I'm fine with:

* Tests / Docs / Docs
* Tests / Docs / Docs (Oldest Sphinx)
* Tests / Docs / Doctest

We can maybe rename that final part later, but for now I prefer to keep words.

(And it's @ambv not `@llanga` on GitHub :)

Reply:
Oh, thanks for correcting me 🤦‍♂️

Reply:
> Yes, sometimes it makes it easier but I'll need to rebase after the separate PR is merged anyway. Plus it makes me easier to smuggle a nice commit message into the resulting thing.

The nice commit message is lost when we squash merge, so don't worry about it. And we can merge `main` into here when the other PR is merged, no need to rebase.

Shall we mark this PR as draft and continue on the other one?

Reply:
> We can maybe rename that final part later, but for now I prefer to keep words.

Note that this specific line refers to the middle part, not the final one.

Reply:
> The nice commit message is lost when we squash merge, so don't worry about it. And we can merge main into here when the other PR is merged, no need to rebase.

Well, it's a habit I cultivate and have to keep doing because habits don't work if you apply them selectively. This is also the reason I don't like squashes since they ruin history in certain ways.

> Shall we mark this PR as draft and continue on the other one?

Yeah, I was considering this. I've been basically waiting for Łukasz to finish his journey back home to resume the work on this. I'll be leaving SLC in about 12 hours myself and probably won't get to the laptop until the Saturday evening (Prague time).